.code-samples.meilisearch.yaml

```diff
@@ -993,3 +993,64 @@ swap_indexes_1: |
         ]
       }
     ]'
+multi_search_1: |-
+  curl \
+    -X POST 'http://localhost:7700/multi-search' \
+    -H 'Content-Type: application/json' \
+    --data-binary '{
+      "queries": [
+        {
+          "indexUid": "movies",
+          "q": "pooh",
+          "limit": 5
+        },
+        {
+          "indexUid": "movies",
+          "q": "nemo",
+          "limit": 5
```

Comment:
Can we show an example of a multi-index search here? This is speakable use case to show off 

e.g `actors` and `movies` indexes